FFMpegCore/FFMPEG/FFBase.cs

```diff
@@ -25,20 +25,14 @@ protected FFBase()
         /// </summary>
         public bool IsWorking
         {
-            get
+             get
             {
-                bool processHasExited;
-
-                try
-                {
-                    processHasExited = Process.HasExited;
-                }
-                catch
+                if (Process != null)
                 {
-                    processHasExited = true;
+                    return !Process.HasExited && Process.GetProcesses().Any(x => x.Id == Process.Id);
                 }
 
-                return !processHasExited && Process.GetProcesses().Any(x => x.Id == Process.Id);
+                return true;
```

Comment:
This should implicitly return `false`. This is why all of the tests currently fail 😬